src/frame.rs

```diff
@@ -773,7 +783,7 @@ impl<R: ReadBytes> FrameReader<R> {
             BlockTime::SampleNumber(snr) => snr,
         };
 
-        let block = Block::new(time, header.block_size as u32, buffer);
+        let block = Block::new(time, header.block_size as u32, header.sample_rate.unwrap_or_default(), buffer);
```

Comment:
`unwrap_or_default` would default to 0 when the header does not specify a sample rate. Instead, we need to get the value from the streaminfo: https://github.com/ruuda/claxon/blob/20fd6a78830ec75918175b2375c21dd667b894ce/src/frame.rs#L193

Reply:
It doesn't seem like streaminfo is available from this function's scope?